app/wama.web/wwwroot/css/site.css

```diff
@@ -78,9 +81,9 @@ td {
 
 /* Sidebar navigation */
 .nav-sidebar {
```

Comment:
I must have added that earlier because it was commented out and all I did was un-comment  it and it came up as a change. I'll get rid of the entire block.